### PR TITLE
[release/6.0] Add RIDs for Tizen6.5 and 7.0

### PR DESCRIPTION
**Pull request by JoonghyunCho**
This PR adds RIDs for
  - the latest Tizen 6.5 Platform version 
  - upcoming Tizen 7.0 Platform version which will be officially released soon in Oct, 2022.

### CODE REVIEW

**Comment by JoonghyunCho** [comment]
@wscho77 @rookiejava @WonyoungChoi Just letting you know about the updates.

**Comment by carlossanlop** [comment]
@wfurt can you please review?

**Review by JongHeonChoi — APPROVED** [review]

**Review by ViktorHofer — APPROVED** [review]

**Review by rookiejava — APPROVED** [review]

File: src/libraries/Microsoft.NETCore.Platforms/src/runtimeGroups.props [hunk 1]
```diff
@@ -232,7 +232,7 @@
     <RuntimeGroup Include="tizen">
       <Parent>linux</Parent>
       <Architectures>x86;armel;arm64</Architectures>
-      <Versions>4.0.0;5.0.0;5.5.0;6.0.0</Versions>
+      <Versions>4.0.0;5.0.0;5.5.0;6.0.0;6.5.0;7.0.0</Versions>
```
Comment (wfurt) [inline src/libraries/Microsoft.NETCore.Platforms/src/runtimeGroups.props:235]: I'm wondering if we really need all the minor versions. We switched to just tracking major in some other cases.

**Review by wfurt — APPROVED** [review]
LGTM.
I left one comment for consideration.

File: src/libraries/Microsoft.NETCore.Platforms/src/runtimeGroups.props (hunk 1 above)
Comment (JoonghyunCho) [reply src/libraries/Microsoft.NETCore.Platforms/src/runtimeGroups.props:235]: Thanks for your comment!
They are all major Tizen Platform versions, but yes I would see if I can remove unnecessary versions on the future changes.

**Comment by wfurt** [comment]
hmm, this is servicing. We may need to update the platform package unless we already have another pending RID change.

**Comment by carlossanlop** [comment]
CI failures are unrelated - They are WASM staging warnings.

Approved, signed-off, OOB package authoring changes included, ready to merge. :shipit:

**Pull request merged**